keyboards/dmqdesign/spin/info.json

```diff
@@ -0,0 +1,28 @@
+{
+    "keyboard_name": "SPIN",
+    "url": "https://www.DMQdesign.com",
+    "maintainer": "Quarren42",
+    "width": 5,
+    "height": 3,
+    "layouts": {
+        "LAYOUT_default": {
+            "key_count": 15,
+            "layout": [
+                { "w": 1, "x": 0, "y": 0 },
+                { "w": 1, "x": 1, "y": 0 },
+                { "w": 1, "x": 2, "y": 0 },
+                { "w": 1, "x": 0, "y": 1 },
+                { "w": 1, "x": 1, "y": 1 },
+                { "w": 1, "x": 2, "y": 1 },
+                { "w": 1, "x": 0, "y": 2 },
+                { "w": 1, "x": 1, "y": 2 },
+                { "w": 1, "x": 2, "y": 2 },
+                { "w": 1, "x": 0, "y": 3 },
+                { "w": 1, "x": 1, "y": 3 },
+                { "w": 1, "x": 2, "y": 3 },
+                { "w": 1, "x": 4, "y": 0 },
+                { "w": 1, "x": 4, "y": 1.5 },
+                { "w": 1, "x": 4, "y": 3 } ]
```

Comment:
Seems to me `k03`, `k13` and `k23` are the push actions for the encoders.

If that statement is correct, then this block is wrong. Please advise.

Reply:
Those statements are correct - had struggled to find the doc for setting up the json but just found it, will fix.